### DISCUSSION TITLE
[Vercel Cron Job] Add user's locale time or specify that it is UTC

### DISCUSSION THREAD

**Discussion by devjiwonchoi** [vercel/community#5954] [Idea]
### Current Behavior

The time is specified as `At 02:30 PM`, which can confuse the users (since it is UTC, not locale) if they haven't read the docs.

![Screenshot of Jiwon Choi's Vercel Cron Jobs](https://github.com/vercel/community/assets/120007119/88c3a1ba-851e-4c49-9eea-1ced00a1f3de)

#### Proposal 1

Modify to the user's locale time.

```sh
At 11:30 PM (Locale Time - KST)
```

#### Proposal 2

Add `(UTC)` next to the time.

```sh
At 02:30 PM (UTC)
```

**Comment by amyegan** [comment]
Thanks for making this suggestion! I'll share it with the team 🙂

**Reply by devjiwonchoi** [reply]
Thank you for considering!

**Reply by amyegan** [reply]
We added a note about UTC time above the Cron Jobs list. Thanks again for suggesting this!

**Reply by devjiwonchoi** [reply]
Thank you!
